webpack/assets/javascripts/react_app/routes/Models/ModelsPage/ModelsPageActions.js

```diff
@@ -0,0 +1,33 @@
+import URI from 'urijs';
+
+import { ajaxRequestAction } from '../../../redux/actions/common';
+import createTableActionTypes from '../../../components/common/table/actionsHelpers/actionTypeCreator';
+
+const controller = 'models'
+
+const tableActionTypes = createTableActionTypes(controller);
+
+import {
+  MODELS_PAGE_LOAD_REQUEST,
+  MODELS_PAGE_LOAD_ERROR,
+  MODELS_PAGE_LOAD_SUCCESS,
+} from '../consts';
+
+export const loadInitialModels = (query = {}) => dispatch => {
+  const url = new URI(`/api/${controller}`);
```

Comment:
Please use the [foremanUrl](https://github.com/theforeman/foreman/blob/c5fbd28993fbe29f3457a329245e8975dd4c7667/webpack/assets/javascripts/foreman_tools.js#L130-L133) helper for URLs